### DISCUSSION TITLE
Prometheus rule can association grafana dashboard?

### DISCUSSION THREAD

**Discussion by taylord0ng** [grafana/grafana#48167] [Dashboards, Panels & Visualizations]
Can prometheus rule association grafana dashboard like grafana alert as grafana dashboard
I create promtheus rule set Dashboard UID and Panel ID
![image](https://user-images.githubusercontent.com/9444710/165065668-edb927df-2bd7-4c92-90e0-eff21cd1bfce.png)
But grafana dashboard alert show nothing
![image](https://user-images.githubusercontent.com/9444710/165065817-a8925242-0db7-4939-b3d3-c016d2326c28.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
